### PR TITLE
KNIDEPLOY-4016

### PR DESCRIPTION
**Pull request by jcfrye77**
Updates to text per https://issues.redhat.com/browse/TELCODOCS-165

### CODE REVIEW

**Comment by jcfrye77** [comment]
@slintes I've made your requested changes. Please review and let me know of any further changes. Thank you.
[https://deploy-preview-31755--osdocs.netlify.app/openshift-enterprise/latest/machine_management/deploying-machine-health-checks.html](url)

**Comment by slintes** [comment]
looks good to me 👍🏼

**Comment by jcfrye77** [comment]
@ahardin-rh Please merge this PR. It is a small update that does not require QE review. Thank you.

File: modules/machine-health-checks-about.adoc [hunk 1]
```diff
@@ -6,32 +6,29 @@
...
-$ oc get machinehealthcheck example -n openshift-machine-api
-----
-[source,terminal]
-----
-NAME      MAXUNHEALTHY   EXPECTEDMACHINES   CURRENTHEALTHY
-example   40%            3                  1
-----
-
-To limit the disruptive impact of machine deletions, the controller drains and deletes only one node at a time. If there are more unhealthy machines than the `maxUnhealthy` threshold allows for in the targeted pool of machines, the controller stops deleting machines and you must manually intervene.
+Consider the timeouts carefully, depending on workloads and requirements.
```
Comment (kalexand-rh) [inline modules/machine-health-checks-about.adoc:24]: It's probably just early, but this line doesn't scan quite right. Consider:

```suggestion
Consider the timeouts carefully, accounting for workloads and requirements.
```

File: modules/machine-health-checks-about.adoc [hunk 2]
```diff
@@ -6,32 +6,29 @@
...
+
+* Long timeouts can result in long periods of downtime for the workload on the unhealthy machine.
+* Too short timeouts can result in a remediation loop. For example, when checking the `NotReady` status, but the machine does not
```
Comment (kalexand-rh) [inline modules/machine-health-checks-about.adoc:27]: The first clause in you example's a fragment, and the sentence would be too long if we formatted it correctly. Please rewrite it to start with an independent clause.

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions and style tweaks, but this change is looking good!

**Comment by jcfrye77** [comment]
@kalexand-rh thanks a million for the helpful review. I've made all the changes, but am having trouble getting the commit to update to this PR. I'm still troubleshooting that now. Many thanks and apologies!

**Comment by jcfrye77** [comment]
@kalexand-rh thanks for your patience. We finally resolved the issue and the commit has now updated the PR. Thanks and apologies!

**Comment by kalexand-rh** [comment]
@jcfrye77, I see you got QE approval on the JIRA. Are you ready for me to merge?

**Comment by jcfrye77** [comment]
@kalexand-rh yes please. It is applicable for both 4.7 and 4.8. Thanks very much.

**Pull request merged**